backend/builder.go

```diff
@@ -150,12 +150,11 @@ func Builder(
 	commitInfo := GetVCSInfo(d, src, version, opts.Enterprise)
 
 	builder = withCue(builder, src).
+		WithDirectory("/src/", src.Directory(""), dagger.ContainerWithDirectoryOpts{
```

Comment:
`src` already is a directory.

```suggestion
		WithDirectory("/src/", src, dagger.ContainerWithDirectoryOpts{
```

Reply:
Good spot 😊